opqueue.go

```diff
@@ -111,11 +106,14 @@ func (q *OpQueue) Enqueue(id ID, op *Op) error {
 		// the condition lock until this method call returns, finishing
 		// its append of the new operation.
 		q.cond.Signal()
-	} else if len(set.Ops()) >= q.width {
+		return nil
+	}
+	if len(set.Ops()) >= q.width {
 		return ErrQueueSaturatedWidth
-	} else {
-		set.append(op)
 	}
+
+	// Q (2023-11) (mh): Why don't we signal here?
```

Comment:
No need to signal: we only need to signal when we add an entry to the map/linked list.

```suggestion
```